### DISCUSSION TITLE
Placement of colon in relation to square brackets at the beginning of a list

### DISCUSSION THREAD

**Discussion by khalid-hussain** [jgm/pandoc#9690] [Q&A] [answered]
Greetings, I have a question. It's better if I show you.

I have the following source:

```
A sample sentence to begin the list.

1. [Test name:]

   This is a sample paragraph.
```

I run the following command:

`pandoc .\pandoc-test.md`

To which I get the following:

```
<p>A sample sentence to begin the list.</p>
<ol type="1">
<li><p>[Test name:]</p>
<p>This is a sample paragraph.</p></li>
</ol>
```

However, if the colon is placed outside the square brackets, i.e. `1. [Test name]:`, I get the following instead:

```
<p>A sample sentence to begin the list.</p>
<ol type="1">
<li>This is a sample paragraph.</li>
</ol>
```

I would like to know if this is by design or is it a bug?

**Comment by jgm** [comment, marked as answer]
By moving the colon you create a reference link definition with empty content.
That's why it doesn't appear in the output.  (You could try adding `[Test name][]` somewhere else and you'll see it turns into a link.
